### PR TITLE
Initial support for yarn

### PR DESCRIPTION
**Pull request by dvalfre**
Install yarn alongside node and npm

### CODE REVIEW

**Comment by dvalfre** [comment]
Provides initual yarn setup, installing it alongside 'node' and 'npm'

* installs the latest version from tarball
* makes it available via `/usr/bin/yarn` and `/usr/bin/yarnpkg`

Limitations

It is not integrated with the deploy process yet.  To use it has to be invoked from deploy_hoks

**Comment by rbankston** [comment]
LGTM

File: cookbooks/node/recipes/yarn.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  source "#{yarn_download_url}"
+  mode 0644
+  backup 0
+end
+
+execute "unarchive Yarn" do
+  cwd "/tmp/yarn"
+  command "tar zxf yarn-latest.tar.gz"
+end
+
```
Comment (radamanthus) [inline cookbooks/node/recipes/yarn.rb]: Maybe add a sleep here to allow the tar to complete before proceeding to the next step?

Comment (dvalfre) [inline cookbooks/node/recipes/yarn.rb]: @radamanthus created https://github.com/engineyard/ey-cookbooks-stable-v5/pull/163 with a newer, improved version of the recipes

**Pull request merged**